department-of-veterans-affairs/caseflow
# Update success messaging for appeal returns to predocket business lines

Resolves [CASEFLOW-6025](https://vajira.max.gov/browse/CASEFLOW-6025)

### Description
These changes correct some slight discrepancies between:
1.  The education pre-docket [designs](https://bahdigital.invisionapp.com/share/QMIAIA7TANS#/screens/296072642?browse) and the actual implementation
2. The education pre-docket and vha pre-docket implementations

### Acceptance Criteria

### Testing Plan
1. open the rails console and create a completed `education_document_search_task` (which also creates the related appeal):
```ruby
FactoryBot.create(:education_document_search_task, :completed)
```
2. as a BVA Intake user, visit their organization queue's Ready for Review tab and select the newly created appeal
3. select the "Return to Education Service"
4. ensure all messaging in the modal refers to "Education Service" in the singular as opposed to "Education Services"
5. Complete the task action and ensure the ensuing success banner matches the designs (see below screenshot):
<img width="1263" alt="Screen Shot 2022-06-03 at 7 29 00 PM" src="https://user-images.githubusercontent.com/18618189/171966576-bd857f4d-77a1-4cb1-9b80-129a2bd98829.png">

6. Repeat steps 1 - 5 for the VHA business line:
```ruby
FactoryBot.create(:vha_document_search_task, :completed)
```
